mne/decoding/transformer.py

```diff
@@ -715,3 +717,164 @@ def transform(self, X):
         X = self.estimator.transform(X)
         X = np.reshape(X.T, [-1, n_epochs, n_times]).transpose([1, 0, 2])
         return X
+
+
+class Filterer(TransformerMixin):
```

Comment:
It is a band pass only if the low and high param are set, so Filter is not appropriate IMO.

Ok for Filter, or TimeFilter to distinguish it from SpatialFilter